wasm/src/test/java/com/dylibso/chicory/wasm/WasmWriterTest.java

```diff
@@ -0,0 +1,31 @@
+package com.dylibso.chicory.wasm;
+
+import static com.dylibso.chicory.wasm.Parser.parseWithoutDecoding;
+import static com.dylibso.chicory.wasm.ParserTest.wasmCorpusFiles;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
copy paste?

Reply:
Good catch. I copied `ParserTest` but it's also not needed there, either.